models/product-projection/README.md

```diff
@@ -0,0 +1,37 @@
+# @commercetools-test-data/product-projection
+
+This package provides the data model for the commercetools platform `ProductProjection` type
+
+https://docs.commercetools.com/api/projects/productProjections
+
+# Install
+
+```bash
+$ pnpm add -D @commercetools-test-data/product-projection
+```
+
+# Usage
+
+```ts
+import {
+  ProductProjection,
+  type TProductProjection,
+  type TProductProjectionRest,
+  type TProductProjectionGraphql,
+} from '@commercetools-test-data/product-projection';
+
+const productProjection =
+  ProductProjection.random().build<TProductProjection>();
+
+// For REST entities
+const productProjectionRest =
+  ProductProjection.random().buildRest<TProductProjection>();
```

Comment:
Shouldn't be?

```suggestion
const productProjectionRest =
  ProductProjection.random().buildRest<TProductProjectionRest>();
```

Reply:
🤦 

Updated here: 6651adb3e276eeb267c33c290b18a519b0177a6d